app/js/views/tag-list.jsx

```diff
@@ -1,25 +1,17 @@
 import React from 'components/react';
 
-import TagItem from 'js/views/tag-item';
-
 export default class TagList extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.foxbox = props.foxbox;
-  }
-
   render() {
-    let tagNodes = this.props.tags.map((tag) => (
-        <TagItem
-          key={tag.id}
-          id={tag.id}
-          name={tag.name}
-          checked={tag.checked}
-          serviceId={this.props.serviceId}
-          foxbox={this.foxbox}/>
-      )
-    );
+    let tagNodes = this.props.tags.map((tag) => {
+      return (<li key={tag} className="tag-list__item">
+        {tag}
+        <button className="tag-list__item-remove"
+                type="button"
+                onClick={this.props.onRemoveTag.bind(null, tag)}>
+          ✕
```

Comment:
> The typo geek in me says that this is the sign for multiplication, not a ballot X

Nope, iirc it's unicode ```cross mark``` [1] :)

> maybe the plus.svg rotated by 45deg

As far as I remember it's too fat and white, but let me check it out.

[1] http://www.fileformat.info/info/unicode/char/274c/index.htm